node/tools/src/config/config_paths.rs

```diff
@@ -17,6 +19,15 @@ impl ConfigPaths {
     /// it as an environment variable. If that also fails, we just use a default value.
     #[instrument(level = "trace")]
     pub(crate) fn resolve(args: &[String]) -> Self {
+        let validator_key = PathSpec {
+            name: "Validator key",
+            flag: "--validator-key",
+            env: "VALIDATOR_KEY",
+            default: "validator_key",
+        }
```

Comment:
Is there a particular reason `clap` isn't used for arg parsing here?

Reply:
nobody bothered to refactor this.